server/src/store.js

```diff
@@ -75,10 +80,40 @@ const handComment = {
 };
 const comments = [foxComment1, foxComment2, handComment];
 
+AWS.config.update({ region: "us-east-2" });

```

Comment:
Nope I refuse to believe this is configured here. Absolutely refuse.

Reply:
Yeah I need to look into whether there's a better place to do this. The examples I was using all did this, which is absurd.